install.sh

```diff
@@ -84,9 +85,6 @@ nvidia-modprobe -u -c0
 # reduces nvidia-smi invocation time 10x from 30 to 2 sec 
 # notable on large VM sizes with multiple GPUs
 # especially when nvidia-smi process is in CPU cgroup
-cp /opt/gpu/nvidia-persistenced.service /etc/systemd/system/nvidia-persistenced.service
-systemctl enable nvidia-persistenced.service
-systemctl restart nvidia-persistenced.service
 nvidia-smi
```

Comment:
we run this command just above ? maybe keep one. ?

Reply:
Discovered that nvidia-smi output is different between the two -- first one right after installer works but the second nvidia-smi fails with `NVIDIA-SMI couldn't find libnvidia-ml.so library in your system. Please make sure that the NVIDIA Display Driver is properly installed and present in your system.`